packages/time-select/src/panel.vue

```diff
@@ -0,0 +1,171 @@
+  if (values.length >= 2) {
+    const hours = parseInt(values[0], 10)
+    const minutes = parseInt(values[1], 10)
+    return {
+      hours,
+      minutes,
+    }
+  }
+  /* istanbul ignore next */
+  return null
+}
+const compareTime = function (time1, time2) {
+  const value1 = parseTime(time1)
+  const value2 = parseTime(time2)
+  const minutes1 = value1.minutes + value1.hours * 60
+  const minutes2 = value2.minutes + value2.hours * 60
+  if (minutes1 === minutes2) {
+    return 0
+  }
+  return minutes1 > minutes2 ? 1 : -1
+}
+const formatTime = function (time) {
+  return (
+    (time.hours < 10 ? '0' + time.hours : time.hours) +
+    ':' +
+    (time.minutes < 10 ? '0' + time.minutes : time.minutes)
+  )
+}
+const nextTime = function (time, step) {
+  const timeValue = parseTime(time)
+  const stepValue = parseTime(step)
+  const next = {
+    hours: timeValue.hours,
+    minutes: timeValue.minutes,
+  }
+  next.minutes += stepValue.minutes
+  next.hours += stepValue.hours
+  next.hours += Math.floor(next.minutes / 60)
+  next.minutes = next.minutes % 60
+  return formatTime(next)
+}
+export default defineComponent({
+  components: { ElScrollbar },
+  emits: ['pick', 'destroy'],
+  setup(props, ctx) {
+    // data
+    const popperClass = ref('')
+    const start = ref('09:00')
+    const end = ref('18:00')
+    const step = ref('00:30')
+    const value = ref('')
+    const defaultValue = ref('')
+    const visible = ref(false)
+    const minTime = ref('')
+    const maxTime = ref('')
+    const width = ref(0)
+    // computed
+    const items = computed(() => {
+      const start = start.value
+      const end = end.value
+      const step = step.value
+      const result = []
+      if (start && end && step) {
+        let current = start
+        while (compareTime(current, end) <= 0) {
+          result.push({
+            value: current,
+            disabled:
+              compareTime(current, minTime.value || '-1:-1') <= 0 ||
+              compareTime(current, maxTime.value || '100:100') >= 0,
+          })
+          current = nextTime(current, step)
+        }
+      }
+      return result
+    })
+    // methods
+    const handleClick = item => {
+      if (!item.disabled) {
+        ctx.$emit('pick', item.value)
+      }
+    }
+    const handleClear = () => {
+      ctx.$emit('pick', null)
+    }
+    const scrollToOption = (selector = '.selected') => {
+      const menu = ctx.$refs.popper.querySelector('.el-picker-panel__content')
+      scrollIntoView(menu, menu.querySelector(selector))
+    },
+    const handleMenuEnter = () => {
+      const selected =
+        items.value.map(item => item.value).indexOf(this.value) !== -1
+      const hasDefault =
+        items.value.map(item => item.value).indexOf(this.defaultValue) !== -1
+      const option =
+        (selected && '.selected') ||
+        (hasDefault && '.default') ||
+        '.time-select-item:not(.disabled)'
+      ctx.$nextTick(() => scrollToOption(option))
```

Comment:
use `nextTick` like this:
`import { nextTick } from 'vue'`
`nextTick(() => scrollToOption(option))`
Since now nextTick is exported as  global API